src/components/UI/atoms/InputText/InputText.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { InputHTMLAttributes } from "react";
+import "./styles.module.css";
+
+type NativeInputProps = InputHTMLAttributes<HTMLInputElement>;
+type CustomProps = {
+  name: InputHTMLAttributes<HTMLInputElement>["name"],
+  label: string,
+  labelClassName?: string | undefined,
+};
+type Props = NativeInputProps & CustomProps;
+
+const defaultLabelClassName = "Form__Label";
+const defaultInputClassName = "Form__Input";
+
+function InputText({
+  name,
+  className,
+  label,
+  labelClassName,
+  ...props
+}: Props) {
+  const labelStyles = !labelClassName
+    ? defaultLabelClassName
+    : `${defaultLabelClassName} ${labelClassName}`;
```

Comment:
You could try something like ```const labelStyles = `${defaultLabelClassName} ${labelClassName || ''})` ```  or ```const labelStyles = `${defaultLabelClassName} ${labelClassName || ''}`.trimEnd()``` here. Not really a big deal, it just could shorten the number of characters you write.